### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**

### CODE REVIEW

File: WeakAuras/Prototypes.lua [hunk 1]
```diff
@@ -5761,7 +5761,7 @@ Private.event_prototypes = {
         trigger.use_cloneId and "true" or "false",
         trigger.use_extend and tonumber(trigger.extend or 0) or 0,
         trigger.use_remaining and "true" or "false",
-        trigger.remaining or 0,
+        type(trigger.remaining) or 0,
```
Comment (mrbuds) [inline WeakAuras/Prototypes.lua:5764]: I don't understand the change
`local triggerRemaining = ` will receive "string" or "number" or 0 if it's nil, or am i wrong?
Did you mean to do `type(trigger.remaining) == "number" and trigger.remaining or 0,` or `tonumber(trigger.remaining) or 0` instead?

Comment (InfusOnWoW) [reply WeakAuras/Prototypes.lua:5764]: Oh yeah.

File: WeakAuras/GenericTrigger.lua [hunk 2]
```diff
@@ -3414,7 +3411,7 @@ do
   end
 
   function WeakAuras.GetDBMStage()
-    return currentStage, currentStageTotal
+    return DBM:GetState()
```
Comment (mrbuds) [inline WeakAuras/GenericTrigger.lua:3414]: I don't see any GetState in my local DBM repo, did you mean GetStage?

Comment (mrbuds) [reply WeakAuras/GenericTrigger.lua:3414]: There is no reason #4483 does not also happen with bigwigs triggers, but their GetStage function is in BossPrototype.lua so i don't know the equivalent to this DBM:GetStage() for them

Comment (InfusOnWoW) [reply WeakAuras/GenericTrigger.lua:3414]: oh right again...

hmm no idea on how to get to that from bigwigs

**Review by mrbuds — APPROVED** [review]

**Comment by InfusOnWoW** [comment]
I'll remove the reverts and also the reporter for the dbm stage trigger claims that the change doesn't work, so I need to investigate that.

**Pull request merged**